examples/bpi-monolith-test-suite/bpi/components/zkcircuits/json/orderCircuitjson.go

```diff
@@ -0,0 +1,19 @@
+package json
+
+type VerifyInput struct {
+	I Input  `json:"I"`
+	P string `json:"P"`
+}
+
+type Input struct {
```

Comment:
let's rename Input to something more descriptive if possible

Reply:
renamed to OrderCircuitInput